src/test/java/teammates/test/cases/action/BaseActionTest.java

```diff
@@ -731,4 +731,7 @@ protected void verifyNumberOfEmailsSent(Action action, int emailCount) {
         assertEquals(emailCount, action.getEmailSender().getEmailsSent().size());
     }
 
+    protected String getPageResultDestination(String userId) {
+        return Const.ViewURIs.INSTRUCTOR_COURSE_DETAILS + "?error=false&user=" + userId;
```

Comment:
This is specific to `InstructorCourseDetailsPageActionTest`. I think this method should be made abstract instead, and implemented in each of the `*ActionTest` classes. Note that the original issue only uses `InstructorCourseDetailsPageActionTest` as an example. It suggests refactoring all the `*ActionTest` classes.